terra-app-helm/aou-sas-chart/templates/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         - name: sasregistry
       serviceAccountName: "{{- .Values.serviceAccount.name }}"
       securityContext:
-        fsGroup: 2000
+        fsGroup: 1001
```

Comment:
how do we know that this is the right group to use?  Is it documented anywhere?

Reply:
no...I looked at the container, and this is what it is